src/oc/web/utils/user.cljs

```diff
@@ -155,3 +156,26 @@
   [current-user-data push-token]
   (let [current-push-tokens (set (:expo-push-tokens current-user-data))]
     (current-push-tokens push-token)))
+
+;; Encode and decode state string for OAuth
+
+(defn- encode-state-string
+  [data]
+  (-> data
+      pr-str
+      js/btoa))
+
+(defn- decode-state-string
+  [s]
+  (-> s
+      js/atob
+      edn/read-string))
```

Comment:
It would be great to move these `encode-state-string` and `decode-state-string` into a `cljc` namespace in lib and use it from both auth and web.
WYT?